devpack/src/main/java/io/neow3j/devpack/contracts/StdLib.java

```diff
@@ -133,6 +133,24 @@
      */
     public static native ByteString base58Decode(String input);
 
+    /**
+     * Encodes the given byte string to a Base58 string. The encoded string contains the checksum
+     * of the binary data.
+     *
+     * @param input The bytes to encode.
+     * @return the encoded string.
+     */
+    public static native String base58CheckEncode(ByteString input);
+
+    /**
+     * Decodes the given Base58-encoded string. The encoded string contains the checksum of the
+     * binary data.
+     *
+     * @param input The Base58-encoded string.
+     * @return the decoded bytes.
+     */
+    public static native String base58CheckDecode(String input);
```

Comment:
I think the return type should be `ByteString`.

Reply:
You're right, I overlooked that.